custom_components/trakt_tv/schema.py

```diff
@@ -65,6 +65,8 @@ def next_to_watch_schema() -> Dict[str, Any]:
         subschemas[trakt_kind.value.identifier] = {
             Required("max_medias", default=3): cv.positive_int,
             Required("exclude", default=[]): list,
+            Required("sort_by", default="released"): In(["released", "title", "last_activity_date", "trakt"]),
```

Comment:
You should not use `SORT_BY_OPTIONS `instead of `In(["released", "title", "last_activity_date", "trakt"])` ?

Reply:
I wasn't sure if all of the options in `SORT_BY_OPTIONS` were valid in the next_to_watch scenario.  I can switch to using it though